examples/githubProfile.ts

```diff
@@ -1,8 +1,12 @@
 import * as request from 'superagent';
 
 export async function getGithubProfile(username: string) {
-	const response = await request.get(`https://api.github.com/users/${username}`);
+    try {
+        const response = await request
+            .get(`https://api.github.com/users/${username}`)
+            .set('User-Agent', 'edorivai/nock-record example');
 
-	const { url, name, location } = response.body;
-	return { url, name, location };
-}
\ No newline at end of file
+        const { url, name, location } = response.body;
+        return { url, name, location };
+    } catch {}
```

Comment:
What's the reason for the try/catch here?

Reply:
Github had made some breaking changes to their API.
Having a `try {} catch {}` is a bit more foolproof? I am happy to remove :)